HW1/P2/P2a.py

```diff
@@ -1,3 +1,30 @@
 from P2 import *
 
-# Your code here
+import findspark
+findspark.init('/Users/george/Documents/spark-1.5.0')
+import pyspark
+import matplotlib.pyplot as plt
+ 
+sc = pyspark.SparkContext()
+
+# crate rdd
+rdd = sc.parallelize(xrange(2000), 10)
```

Comment:
I believe doing it with 10 is correct because once you perform cartesian in the next line, it will create 100 partitions (I printed it to test this). This also seems to be consistent with most of the other submissions.